src/main/java/seedu/duke/Event.java

```diff
@@ -0,0 +1,33 @@
+package seedu.duke;
+
+import java.util.HashMap;
+
+/**
+ * Represents a task that has a start date and an end date
+ */
+public class Event extends Task {
+
+    private String from;
+    private String to;
+
+    /**
+     *
+     * @param parsed A hashmap containing the event name, from date, and to date
+     * @throws DukeException Exception thrown when event name is empty
+     */
```

Comment:
Is there a reason why the constructor header comment is not included in this javadoc?